secure/key-manager/Cargo.toml

```diff
@@ -10,6 +10,8 @@ license = "Apache-2.0"
 publish = false
 
 [dependencies]
+anyhow = "1.0.38"
```

Comment:
- We generally try to avoid the anyhow dependency in the TCB (e.g., key manager, secure storage and safety rules). Is there a reason we need this in the client code? Perhaps we could adopt a simpler error structure in the client? 
- Ideally we wouldn't need to do the bcs parsing in the key manager, but I see why it's now being done there..

Reply:
1. I'll fix that (and just use a Box<dyn Error>) although the TCB does have a dep on anyhow regardless of this change due to its dependencies.
2. agreed, theres still a lot to do to improve these interfaces and this is just the first step.

Reply:
what's wrong with `anyhow` btw?